crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -727,12 +728,17 @@ fn spawn_broadcast_node_annoucements(
     alias: &str,
     announcement_addresses: Vec<NetAddress>,
     peer_manager: Arc<PeerManager>,
+    channel_manager: Arc<ChannelManager>,
 ) -> Result<RemoteHandle<()>> {
     let alias = alias_as_bytes(alias)?;
     let (fut, remote_handle) = async move {
         let mut interval = tokio::time::interval(BROADCAST_NODE_ANNOUNCEMENT_INTERVAL);
         loop {
-            broadcast_node_announcement(&peer_manager, alias, announcement_addresses.clone());
+            if !channel_manager.list_channels().is_empty() {
```

Comment:
@bonomat I think we should filter here for only public channels.

Reply:
You might be right... I'll change it

Reply:
Done